<launch>

    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true">
        <rosparam file="$(find roborts_nav)/param/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find roborts_nav)/param/costmap_common_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find roborts_nav)/param/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find roborts_nav)/param/global_costmap_params.yaml" command="load" />
        

        <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS" />
        <param name="base_global_planner" value="global_planner/GlobalPlanner" />
        <rosparam file="$(find roborts_nav)/param/teb_local_planner_params.yaml" command="load" />
        <rosparam file="$(find roborts_nav)/param/global_planner_params.yaml" command="load" />
    </node>

</launch>